ct/bookstack.sh

```diff
@@ -60,20 +60,31 @@ check_container_resources
 if [[ ! -d /opt/bookstack ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
 RELEASE=$(curl -s https://api.github.com/repos/BookStackApp/BookStack/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
 if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+
   msg_info "Stopping Apache2"
-  systemctl stop apache2
+  systemctl stop apache2 
   msg_ok "Services Stopped"
 
   msg_info "Updating ${APP} to ${RELEASE}"
-  cp /opt/bookstack/.env /opt/.env
+  mv /opt/bookstack/ /opt/bookstack-backup 
+  tar -czf /opt/bookstack-backup_${RELEASE}.tar.gz /opt/bookstack/.env /opt/bookstack/public/uploads /opt/bookstack/storage/uploads /opt/bookstack/themes &>/dev/null
```

Comment:
```suggestion
```
I don't think we should start mixing actual backup strategies with zip files and dumps into our update scripts. A temporary backup folder while updating which is used for migrating and is removed afterwards is enough for our scope imo. Actual backups should be handles by users themselves, by having LXC backups/snaphots.

Reply:
Yeah you a prob right.  I just stuck to the guide there and thought it looks like a good idea to implement that. Maybe I think about this a bit more and start a discussion about this topic.

I will also remove the sql dump then.